engineering/definition-of-ready.md

```diff
@@ -30,7 +30,19 @@ The following non-exhaustive initial list of the different kinds of tasks that t
 The following describes what is the information needed for a member of the engineering team to start working on a specific type of work item.
 
 (ready:feature-enablement)=
-### Feature enablement
+### Feature Enablement
```

Comment:
```suggestion
### Feature enablement
```

On this page words in headings are not capitalized, not sure about other pages in general.